documentation/src/main/asciidoc/topics/proc_spring_boot_serialization.adoc

```diff
@@ -1,16 +1,34 @@
+[id='spring-boot-serialization_{context}']
 = Configuring Marshalling
-Configure {brandname} servers to use Java serialization to marshall objects.
+Configure {brandname} to marshall Java objects into binary format so they can be transferred over the wire or stored to disk.
 
-By default {brandname} server uses a ProtoStream serialization library as the
-default marshaller. However, the ProtoStream marshaller is not supported for
-Spring integration. For this reason you should use the Java Serialization Marshaller.
+By default {brandname} uses a Java Serialization marshaller, which requires you to add your classes to an allow list.
+As an alternative you can use ProtoStream.
```

Comment:
ProtoStream also requires annotating key/value classes with protostream annotations and adding the protostream-processor dependency. Maybe we should link to the protostream marshalling chapter in the dev guide.

Reply:
Added to the new PR: https://github.com/infinispan/infinispan/pull/9217/files#diff-1fea95fa62e0399a3e9694ddc76b8da26a5d7a1861f6d10b89591350cdc1ed84R6